lib/interfaces/src/IOExpander.cpp

```diff
@@ -0,0 +1,29 @@
+#include "IOExpander.h"
+#include "SharedFirmwareTypes.h"
+
+/* Returns intended watchdog state */
+IOExpander::IOExpander() : mcp(MCP23017(0x20))
```

Comment:
I'd prefer if this 0x20 was an argument instead of a constant. This should be passed into the constructor.